source/dagger/gameplay/common/aiming_system.cpp

```diff
@@ -0,0 +1,70 @@
+#define _USE_MATH_DEFINES
+
+#include "aiming_system.h"
+
+#include "core/graphics/sprite.h"
+#include "core/input/inputs.h"
+
+#include <cmath>
+#include <math.h>
+
+void AimingSystem::OnInitialize(Registry& registry_, Entity entity_)
+{
+    InputReceiver& receiver = registry_.get<InputReceiver>(entity_);
```

Comment:
@gavrilovmiroslav , do we still need this?

Reply:
No, this was changed recently, in the same event that broke Mac. We no longer need this, but the `Get` function needs to be used from `InputReceiver`. So: no setting commands to 0 manually here